docs/features/openmdao/openmdao_basics.rst

```diff
@@ -26,8 +26,9 @@ Pay attention to these four class methods which are very common in OpenMDAO mode
     - The `compute` method establishes how the outputs should be computed.
     - The `compute_partials` method tells the component how to compute partial derivatives.
```

Comment:
If these are `` instead of ` it will make an inline code block instead of italics, which might stand out better in the docs.

Reply:
Good point, I'll change them